src/components/token-screen-content-container/token-screen-content-container.styles.ts

```diff
@@ -20,16 +21,12 @@ export const useTokenScreenContentContainerStyles = createUseStyles(({ colors, t
     justifyContent: 'space-between',
     alignItems: 'center',
     paddingVertical: formatSize(4),
-    paddingHorizontal: formatSize(8),
+    paddingHorizontal: formatSize(6),
     borderRadius: formatSize(14),
     backgroundColor: colors.blue
   },
   delegateText: {
     ...typography.caption13Semibold,
-    color: colors.white
-  },
-  apyText: {
-    ...typography.caption13Semibold,
-    color: colors.white
+    color: basicLightColors.white
```

Comment:
why we can't use colors object here?

Reply:
and in other places where you have these changes.

Reply:
because colors object change colors by themes (dark/light). and colors.white will change on the dark theme to black color but we need always white color with any themes